inmemory.go

```diff
@@ -0,0 +1,82 @@
+package repohandler
+
+import (
+	"errors"
+
+	"github.com/google/uuid"
+)
+
+// InMemoryDB implements RepoHandler for a custom in memory database
+type InMemoryDB struct {
+	DB *map[string]interface{}
```

Comment:
use lower case/unexport db as we are doing transaction using methods

`+`

use specific struct than a general one.